lib/module-declaration.js

```diff
@@ -125,7 +125,8 @@ const generateModuleDeclaration = (module, index, API) => {
     if (moduleMethod.returns) {
       returnType = moduleMethod.returns
       // Account for methods on the process module that return a custom type/structure, we need to reference the Electron namespace to use these types
-      if (module.name === 'process' && moduleMethod.returns.type !== 'Object') {
+      if (module.name === 'process' && moduleMethod.returns.type !== 'Object' &&
+          typeof moduleMethod.returns.type === 'string') {
```

Comment:
Asking for my own understanding, could you explain what this addition does wrt the `Number | null` documentation change?

Reply:
Previously all modifications to the process API in nodejs used complex structures as the return values.  These lived in the "Electron" namespace so we blanket prefixed all the type names here so that the types were referenced correctly.

This PR adds "number" and "null" as return types which are primitives and don't exist in the electron namespace.  Primitives are still type objects at this point in the code not string names.  So we can just typeof check to see if we should prefix with "Electron" or not